src/app/shared/object-detail/object-detail.component.scss

```diff
@@ -0,0 +1,28 @@
+@import '../../../styles/variables';
+@import '../../../styles/mixins';
+
+ds-wrapper-detail-element ::ng-deep {
+  div.thumbnail > img {
+    height: $card-thumbnail-height;
+    width: 100%;
+  }
+  .card-title {
+    line-height: $headings-line-height;
+    height: ($headings-line-height*3) +em;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+  .item-abstract {
+    line-height: $line-height-base;
+    height: ($line-height-base*5)+em;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+  .item-authors{
+    line-height: $line-height-base;
+    height: ($line-height-base*1.5)+em;
+  }
+  div.card {
+    margin-bottom: 20px;
```

Comment:
Please use a sass variable, or better yet [a bootstrap spacing CSS class](https://getbootstrap.com/docs/4.0/utilities/spacing/)

Reply:
removed because is not used